client/extensions/woocommerce/app/shipping/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import ShippingOrigin from './shipping-origin';
+import ShippingZoneList from './shipping-zone-list';
+import ShippingLabels from './shipping-labels';
+import ShippingPackageList from './shipping-package-list';
+
+class Shipping extends Component {
+	render() {
+		return (
+			<Main className="woocommerce shipping">
```

Comment:
After you rebase, please modify Shipping to accept a className property -- and it will receive it from the WooCommerce "controller" - https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/index.js#L147